mypy/main.py

```diff
@@ -292,6 +283,7 @@ def usage(msg: str = None) -> None:
   --python-version x.y  use Python x.y
   --silent, --silent-imports  don't follow imports to .py files
   -f, --dirty-stubs  don't warn if typeshed is out of sync
+  --implicit-any     behave as though all functions were annotated with Any
```

Comment:
I wouldn't document this feature yet, or I'd mark this as experimental, as this is probably not the final user interface for this in case we decide to incorporate this in the longer term -- likely this would become how mypy behaves by default, and a strict/strong mode would be a file-level option.